shell.nix

```diff
@@ -1,31 +1,21 @@
 # Credit to https://github.com/profunktor/redis4cats/blob/master/shell.nix
 let
-  config = {
-    packageOverrides = pkgs: rec {
-      sbt = pkgs.sbt.overrideAttrs (
-        old: rec {
-          version = "1.3.13";
-
-          patchPhase = ''
-            echo -java-home ${pkgs.openjdk11} >> conf/sbtopts
-          '';
-        }
-      );
-    };
+  jdk11 = self: super: rec {
+    jre = pkgs.jdk11;
   };
 
   nixpkgs = fetchTarball {
-    name   = "NixOS-unstable-08-06-2020";
-    url    = "https://github.com/NixOS/nixpkgs-channels/archive/dcb64ea42e6.tar.gz";
-    sha256 = "0i77sgs0gic6pwbkvk9lbpfshgizdrqyh18law2ji1409azc09w0";
+    name = "nixpkgs-20-09";
+    url = "https://github.com/NixOS/nixpkgs/archive/20.09.tar.gz";
```

Comment:
This is fine for `jdk11` but why not staying on `unstable`?

Reply:
if there's no need to use newer packages I'd stick to a stable release :)

Reply:
I assume this identifier is as stable as specifying a sha1?

Reply:
I think so :) it's not exactly a tag i think, but i doubt they change past "releases". Worst case we find out when there's a conflict with the sha256 below.